packages/tldraw/src/lib/tools/SelectTool/childStates/Brushing.ts

```diff
@@ -180,7 +180,7 @@ export class Brushing extends StateNode {
 		}
 
 		this.editor.updateInstanceState({ brush: { ...this.brush.toJson() } })
-		this.editor.setSelectedShapes(Array.from(results), { squashing: true })
+		this.editor.setSelectedShapes(Array.from(results).sort(), { squashing: true })
```

Comment:
Me and @ds300 discovered some react thrashing of selection indicator component when using brush selecting. It was caused by updating the selected shapes and not using a sorted array.